config/webpack/webpack.config.js

```diff
@@ -1,6 +1,8 @@
 const { webpackConfig: baseWebpackConfig, merge } = require('shakapacker')
+const ReactRefreshWebpackPlugin = require('@pmmmwh/react-refresh-webpack-plugin');
```

Comment:
@jcoyne maybe I am missing something but -- do we need this in package.json?

Reply:
It's already here: https://github.com/sul-dlss/sul-embed/blob/master/package.json#L67